.github/workflows/release-please.yaml

```diff
@@ -15,3 +15,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: googleapis/release-please-action@a02a34c4d625f9be7cb89156071d8567266a2445 # v4.2.0
+
+  publish-helm:
+    needs: release-please
+    runs-on: ubuntu-latest
+    steps:
+      - uses: azure/setup-helm@v4.2.0
+        with:
+          version: v3.17.0
+      - run: |
+          helm package charts/prometheus-multipathd-exporter --destination .
+      - run: |
+          echo "${{ secrets.GITHUB_TOKEN }}" | helm registry login ghcr.io --username ${{ github.actor }} --password-stdin
+      - run: |
+          helm push prometheus-multipathd-exporter-*.tgz oci://ghcr.io/${{ github.repository_owner }}/charts
+
```

Comment:
https://github.com/vexxhost/multipathd_exporter/blob/main/release-please-config.json#L4-L6

You need to add the path for the chart to `release-type` set to `helm`.  This way, it'll automatically bump the version in the `release-please` PR, so when we merge, it'll have the updated version.

Also, you would want to use the release-please-action conditional if a new release is done to run this.   Let's use this GitHub Action so things can stay consistent and reusable:

https://github.com/marketplace/actions/helm-oci-chart-releaser